src/iodide-api/resolve.js

```diff
@@ -0,0 +1,23 @@
+
+window.iodideRequireExplicitResolution = false
+window.iodideRequireExplicitResolutionStatus = undefined
+
+const expectResolution = () => {
+  window.iodideRequireExplicitResolution = true
+  window.iodideRequireExplicitResolutionStatus = 'pending'
+  window.iodideExplicitResolver = new Promise((resolve) => {
```

Comment:
Yep, I wasn't sure if this was the right way to do it, but figured I'd give it a shot and see who'd object to it :) the `iodideExplicitResolver` is a temporary Promise that polls for a flag change on `window.iodideRequireExplicitResolutionStatus`. I thought about adding all of this directly in `actions.js` but figured this would be worth attempting. The premise here is that an `eval` code block needs to set some flag saying it will wait for a flag to be set via `iodide.resolve`, and once it is, it will resolve the Promise in `window.iodideExplicitResolver`.

This can be done in the state in some way - using `window` does feel a little funky here - but to be clear we just need this temporary Promise for the purposes of a single `evaluateCodeCell` call that doesn't require me to import `store` into `resolve.js`. If that makes more sense I can do that. `iodideGlobalEvalDequeuer` is not accurate here, since the global eval queue Promise is in `actions.js` (line 16, `evaluationQueue`).

I think there are a few ways of handling this, and maybe it'd be good to tackle this in front of a whiteboard tomorrow so we can get it right.